### PR TITLE
Create readonly user and initial dashboard at startup

### PR DESCRIPTION
**Pull request by adaszko**

### CODE REVIEW

File: influxdb/00-init.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```
Comment (cloudhead) [inline influxdb/00-init.sh:1]:
```suggestion
#!/bin/sh
```

File: influxdb/template.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
+                  |> filter(fn: (r) => r["_measurement"] == "membership" or r["_measurement"] == "peers")
+                  |> filter(fn: (r) => r["_field"] == "active" or r["_field"] == "connected" or r["_field"] == "passive")
+                  |> filter(fn: (r) => r["peer_id"] == "hyyqpngdoe4x4oto3emfdppbw7sj1pfaghbpmmhz5rqiuqg8uofmeo")
+                  |> aggregateWindow(every: v.windowPeriod, fn: last, createEmpty: false)
+                  |> yield(name: "last")
+        width: 5
+        xCol: _time
+        yCol: _value
+    name: org-node
```
Comment (cloudhead) [inline influxdb/template.yaml:58]: Newline?

File: influxdb/00-init.sh (hunk 1 above)
Comment (adaszko) [reply influxdb/00-init.sh:1]: I'm using various bash-specific options for shell script robustness:
```
set -o errexit
set -o nounset
set -o pipefail
set -o errtrace
```
They aren't available in regular sh.

File: influxdb/template.yaml (hunk 2 above)
Comment (adaszko) [reply influxdb/template.yaml]: Added

File: influxdb/00-init.sh (hunk 1 above)
Comment (cloudhead) [reply influxdb/00-init.sh:1]: Okk gotcha.

**Comment by cloudhead** [comment]
Rebased and merged in c095d42a5b5c4850c18a97aa6f1d67459e160ea8

**Pull request closed without merging**